lumbermill-aws/src/main/java/lumbermill/S3.java

```diff
@@ -40,6 +38,39 @@
     private Codec<E> defaultEntityCodec = (Codec<E>) Codecs.BYTES;
 
 
+     /**
+      * Polls an S3 bucket for newly created files
+      * <p>
+      * 'bucket', 'prefix' and 'maxConcurrent' are mandatory,
+      * 'notOlderThanInMins' and 'roleArn' are optional
+      *
+      * <pre>
+      * Groovy usage:
+      *  {@code
+      * s3.poll (
+      *     bucket:              'the_bucket',
+      *     prefix:              'the_prefix',
+      *     suffix:              'the_suffix',
+      *     maxConcurrent:       1,
+      *     notOlderThanInMins:  60,
+      *     roleArn:             'the_role_arn'
+      * ).onFile(f -> f.doOnNext(console.stdout())
+      * }
+      * </pre>
+      */
+     public Poll poll(Map<String, Object> config) {
+         MapWrap conf = MapWrap.of(config).assertExists("bucket", "prefix", "maxConcurrent");
+         String bucket   = conf.asStringTemplate("bucket").format().get();
+         String prefix   = conf.asStringTemplate("prefix").format().get();
+         String suffix   = conf.asString("suffix", "");
```

Comment:
Is it possible to use asStringTemplate with a default value?

The line below resulted in a default of the string `"null"` instead of empty string.
`String suffix   = conf.asStringTemplate("suffix").format().orElse("");`